tests/v2/unit/adapters/torch/lightning/test_helpers.py

```diff
@@ -15,13 +20,11 @@
     AnnotationSceneKind,
 )
 from otx.v2.api.entities.color import Color
-from otx.v2.api.entities.dataset_item import DatasetItemEntity
-from otx.v2.api.entities.datasets import DatasetEntity
 from otx.v2.api.entities.image import Image
 from otx.v2.api.entities.label import Domain, LabelEntity
 from otx.v2.api.entities.label_schema import LabelGroup, LabelGroupType, LabelSchemaEntity
 from otx.v2.api.entities.shapes.ellipse import Ellipse
-from otx.v2.api.entities.shapes.polygon import Point, Polygon
+from otx.v2.api.entities.shapes.polygon import Polygon
```

Comment:
Would it be possible to use datumaro `Polygon` here?

Reply:
Unfortunately, not for now. I want to reuse the `generate_random_annotated_image` method at this moment, which generates random otx.api.entities. I will modify this to create random datumaro dataset item later.